spec/author_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../author'
+
+describe Author do
+  it 'should have last_name' do
+    author = Author.new('John', 'Doe')
+    expect(author.last_name).to eq('Doe')
+  end
+
+  it 'should have first_name' do
+    author = Author.new('John', 'Doe')
+    expect(author.first_name).to eq('John')
+  end
+end
```

Comment:
- [ ] As stated in the requirements, all methods inside classes should be tested. So please ensure to add tests for the `add_item` methods from the `Author` class. 👌

### Requirement :point_down: 
![Screenshot from 2023-11-03 14-59-27](https://github.com/dvcorreia1984/ruby-group-capstone/assets/89228743/62dd0196-4c4f-4871-86a0-c8f997697f3b)

Reply:
Thank you Ridwanediallo, I added the add_item method test.